src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -255,11 +268,42 @@ private void readRcaEnabledFromConf() {
         });
   }
 
+  /**
+   * Reads the mutedRCAs value from the conf file.
+   */
+  private void readMutedRcasFromConf() {
+    // all the config filed are located in same location as RCA_ENABLED_CONF_LOCATION
+    Path filePath = Paths.get(RCA_ENABLED_CONF_LOCATION, MUTED_RCAS_CONF_FILE);
+
+    Util.invokePrivileged(
+            () -> {
+              try (Scanner sc = new Scanner(filePath)) {
+                mutedRcas = sc.nextLine();
```

Comment:
Once we read the RCAs, we should also proess the string and make sure that all the RCAs mentioned are valid graph nodes. If they are not, then we should write an error log and discard that element.

Reply:
Agreed, made the change in latest revision